### DISCUSSION TITLE
2023 Open Visualization Collaborator Summit (September 6-7, New York City)

### DISCUSSION THREAD

**Discussion by Pessimistress** [visgl/deck.gl#7928] [Announcement]
The OpenJS Foundation and Foursquare will host the 2023 Open Visualization Collaborator Summit on September 6-7 in Foursquare's New York City office.

Join us as we bring together an international audience of geospatial minds to discuss how they are using deck.gl to build apps, foster more contributions, and envisage the future of the leading open-source mapping library. Meet developers passionate about building visualization applications, share your own amazing work, meet others facing similar challenges, and discuss the community's future.

Unfortunately, we will not be able to provide a live stream, however a recording will be available after the event.

Reserve your spot now:
https://deck.gl/events/new-york-summit-2023/

**Comment by wilson0028** [comment]
Hello,

Do you know where the recordings are for this event?  I can't seem to find them anywhere.  Thank you!

**Reply by chrisgervang** [reply]
Hey @wilson0028, we've uploaded all of the recordings to [this playlist](https://www.youtube.com/watch?v=wNKB67SXukk&list=PLyspMSh4XhLOgJgtNz1bpbaW7mpNbqCQy&pp=iAQB) on the OpenJS Foundation youtube channel. Enjoy!
